frappe/integrations/doctype/token_cache/token_cache.py

```diff
@@ -52,8 +52,8 @@ def update_data(self, data):
 		return self
 
 	def get_expires_in(self):
-		expiry_time = frappe.utils.get_datetime(self.modified) + timedelta(self.expires_in)
-		return (datetime.now() - expiry_time).total_seconds()
+		expiry_time = frappe.utils.get_datetime(self.modified) + timedelta(seconds=self.expires_in)
```

Comment:
Isn't it `days=self.expires_in` ?

Reply:
NO
[https://www.oauth.com/oauth2-servers/making-authenticated-requests/refreshing-an-access-token/](https://www.oauth.com/oauth2-servers/making-authenticated-requests/refreshing-an-access-token/)

A simple google search would have answered this question